sdks/wasm/bindings.cs

```diff
@@ -119,20 +124,29 @@ static int UnBindRawJSObjectAndFree(int gcHandle)
                     raw_to_js.Remove(raw_obj);
                 }
 
-
                 obj.Dispose();
 
                 var gCHandle = obj.Handle;
                 obj.Handle.Free();
                 obj.JSHandle = -1;
                 obj.RawObject = null;
+                obj = null;
```

Comment:
Why are these null assignments before return needed? Are they working around a runtime bug?

Reply:
Just wanted to make sure things were clean.